docs/evaluate/development-production-features/temporal-nexus.mdx

```diff
@@ -17,32 +17,96 @@ keywords:
+The result is an integrated Temporal experience with improved security, observability, and reliability for cross-team, cross-domain, cross-namespace, and multi-region use cases.
+
+**Why use Temporal Nexus?**
+
+Temporal Nexus provides:
+
+- **A fully integrated Temporal SDK experience**:
+  - Use Nexus Services from a caller Workflow.
+  - Run Nexus Services in a Worker, often the same Worker as underlying primitives.
+  - Implement long-running asynchronous Nexus Operations as Workflows.
+  - Handle low-latency synchronous Nexus Operations with Temporal primitives or arbitrary code.
+  - At-least-once execution. Option for at-most-once execution using handler Workflow policy.
+- **Nexus Endpoints with a queue-based Worker architecture**:
+  - Connect callers and handlers through Nexus Endpoints, a reverse proxy for Nexus Services.
+  - Manage Endpoints in the Temporal Nexus Registry.
+  - Use an Nexus Endpoint by name, which routes to an upstream target Namespace and Task Queue.
+  - Workers poll an Endpoint's target Task Queue, for automatic load balancing.
+- **Built-in Temporal Nexus Machinery**:
+  - Reliable state-machine-based invocation and completion callbacks for Nexus Operations.
+  - Atomic handoff from Workflow Event History to Nexus Operation state machines.
+  - Automatic retries and circuit breaking.
+- **Integrated observability**:
+  - Operation lifecycle and error info in Workflow Event History.
+  - Bi-directional linking for end-to-end debugging across Namespaces.
+  - Metrics, traces, and logs.
+- **Improved blast radius isolation**:
+  - Each team can have its own Namespace.
+  - Per-Namespace security & rate limits.
+  - Isolate misbehaving Workers.
+- **Enhanced security and connectivity in Temporal Cloud**
+  - Built-in access controls. Restrict which callers can use a Nexus Endpoint.
+  - Global mTLS-secured Envoy mesh, connecting all Namespaces in an Account.
+
+**Should you be using Temporal Nexus?**
+
+- **Cross-team, cross-domain, and cross-namespace use cases** \-
+Nexus is purpose-built to connect Temporal Applications within and across Namespaces.
+It addresses the limitations of Child Workflows and Activity Wrappers that target a remote Namespace.
+Nexus has a streamlined Temporal developer experience, reliable execution, and integrated observability.
+
+- **Share a subset of a Temporal Application** \-
+Abstract and share a subset of an Application as a Nexus Service.
+Expose Services on a Nexus Endpoint for others to use and secure access with Endpoint access control policies.
+Nexus Endpoints decouple callers from handlers, so teams can operate more autonomously.
 
-For a deep dive into Temporal Nexus, see the [Temporal Nexus Encyclopedia section](https://docs.temporal.io/nexus).
-To connect with the Nexus community, join the [#nexus](https://temporalio.slack.com/archives/C07LQN0JK9B) channel in [Temporal Slack](https://t.mp/slack).
+- **Design for growth** \-
```

Comment:
82-84 is really strong.

BTW, will readers know what a "service consumer" is in this context?

Reply:
Thanks! Yes IMO all app devs should know what a service consumer is. 👍